apps/users/forms.py

```diff
@@ -30,9 +30,18 @@ class UploadAvatarForm(forms.Form):
 
 
 class ApiKeyForm(forms.ModelForm):
+    read_only = forms.BooleanField(
+        required=False,
+        initial=True,
+        label=gettext("Read Only - Can only view/fetch data"),
+        help_text=gettext("Leave unchecked for Read/Write access. "
+                          "This cannot be changed after the key is created."),
+        widget=forms.CheckboxInput(),
+    )
```

Comment:
Thinking about this now I want to invert the condition on the UI so that by default it is unchecked (read only) and the user must check the box to allow write access. 

I just think that makes more sense from a UI perspective and gives a clearer signal to the user that they are allowing something extra by checking the box.